build/NuSpecs/Microsoft.WindowsAppSDK.Bootstrap.targets

```diff
@@ -5,10 +5,13 @@
   <PropertyGroup>
     <_WindowsAppSDKFoundationPlatform Condition="'$(Platform)' == 'Win32'">x86</_WindowsAppSDKFoundationPlatform>
     <_WindowsAppSDKFoundationPlatform Condition="'$(Platform)' != 'Win32'">$(Platform)</_WindowsAppSDKFoundationPlatform>
+    <_WindowsAppSDKBootstrapPlatform Condition="'$(_WindowsAppSDKFoundationPlatform)' == 'AnyCPU'">x86</_WindowsAppSDKBootstrapPlatform>
```

Comment:
in WindowsAppSDK-Nuget-Native.C.props, we are still trying to copy from path based on _WindowsAppSDKFoundationPlatform.

Reply:
Wouldn't it make more sense to make it empty (or add !AnyCPU in line 7) so it isn't set at all for anycpu, and then deal with it not set?

Reply:
Native.C is only used by C/C++ projects, where AnyCPU doesn't exist?

Reply:
> make it empty (or add !AnyCPU in line 7) so it isn't set at all for anycpu, and then deal with it not set?

It could also be handled that way, but the conditional logic gets a bit messier (thinking long-term as well  as today).

This defines _WindowsAppSDKBootstrapPlatform as the key subdir to grab from, which 1-1 matches _WindowsappSDKFoundationPlatform except when building AnyCPU. So we precompute the answer here and then just use it below. That's a tad cleaner than using undefined/"" for AnyCPU and then computing it below.

I was careful to define a Bootstrap-specific property instead of trying to rely on / change the Foundation one, to avoid unwelcome entanglements down the road.

I tried to make the copy entirely overrideable e.g. msbuild /p:WindowsAppSDKBootstrapCopy=false to disable the copy, =true to do it and "" to "do the default", but it got a bit twisty and busted. I probably botched the logic but wasn't clearing up quickly so not worth the time sink. Not required today so I deferred that degree of control to a future date if/when desired. But this precompute-then-just-use leaves open the door to (more easily) go there, if/when desired